_tour/basics.md

```diff
@@ -205,21 +202,23 @@ We will cover classes in depth [later](classes.html).
 
 ## Case Classes
 
-Scala has a special type of class called a "case" class.  By default, case classes are immutable and compared by value. You can define case classes with the `case class` keywords.
+Scala has a special type of class called a "case" class. By default, case classes are immutable, and they are compared by value (unlike classes, which are compared by reference). This makes them particularly useful for [pattern matching](https://docs.scala-lang.org/tour/pattern-matching.html#matching-on-case-classes).
```

Comment:
This isn't what makes them useful for pattern matching.  They are _additionally_ useful for pattern matching because the compiler automatically generates the matcher.

Reply:
I think I'm reading this as "for pattern matching, they are particularly useful", and you are reading this as "they are useful specifically for pattern matching", so I will change this due to ambiguity, thanks!

Reply:
I am reading it as, "The qualities I described above _is what makes them useful for matching."  This is what I object to, because it mostly isn't the reason they're useful for matching.